oss-internship-2020/guetzli/guetzli_transaction.h

```diff
@@ -0,0 +1,62 @@
+#pragma once
+
+#include <libgen.h>
+#include <syscall.h>
+
+#include "guetzli_sandbox.h"
+#include "sandboxed_api/transaction.h"
+#include "sandboxed_api/vars.h"
+
+namespace guetzli {
+namespace sandbox {
+
+constexpr int kDefaultTransactionRetryCount = 0;
+constexpr uint64_t kMpixPixels = 1'000'000;
+
+enum class ImageType {
+  JPEG,
```

Comment:
constants and enum values should be `kConstantValue` named
https://google.github.io/styleguide/cppguide.html#Constant_Names